conda_build/build.py

```diff
@@ -4,6 +4,8 @@
 Module that does most of the heavy lifting for the ``conda build`` command.
 """
 
+from __future__ import annotations
```

Comment:
You might want to consider adding this to every file you've touched. It gets very difficult to track which annotations are available in what version. It's icky, but might prevent future headaches.

Reply:
All files included here either already have this or it was added

Reply:
Gah, some of the tops of files got truncated in the preview. Carry on!